src/main/proto/service.proto

```diff
@@ -16,26 +16,28 @@ syntax = "proto2";
 
 package com.google.finapp;
 
-import "google/protobuf/empty.proto";
-
 option java_multiple_files = true;
 option java_package = "com.google.finapp";
 option java_outer_classname = "ServiceProto";
 
 service FinApp {
 
-  rpc CreateCustomer(Customer) returns (.google.protobuf.Empty) {}
+  rpc CreateCustomer(Customer) returns (CreateCustomerResponse) {}
```

Comment:
Actually, can we standardize all request and response names? As in for the method `CreateCustomer` we would have `CreateCustomerRequest` (instead of just `Customer`) and `CreateCustomerResponse`. 

My reasoning is that future methods' requests can't be as easily simplified down to things like `Customer` or `Account`. For example, `MoveAccountBalance` requires 2 account ids and an amount (and is different from a "Transaction"). I will most likely label the request `MoveAccountBalanceRequest`, and I think it will be easier for users to understand and navigate this file if there are directly labeled request and responses for each RPC.

The downside to this is lack of reusability if there are multiple methods that share a request signature.

I'm not sure what the preferred nomenclature is for gRPC, but I think this would clarify things a bit.

Thoughts?

Reply:
Using {Method}(Request|Response) is indeed standard practice and it makes it easier to evolve such protos in the long-run. I was thinking to propose it but decided against it for simplicity.

However, it seems you both like it, so go for it :) (with this or separate PR up to you)